packages/google_fonts/generator/README.md

```diff
@@ -1,7 +1,10 @@
-Generates the `GoogleFonts` class, list of supported families.
+If you notice fonts that are on [fonts.google.com](https://fonts.google.com) that do not appear in
```

Comment:
If someone isn't already familiar with maintaining this package, they aren't going to know to go find generator/README.md to find this. Having package-specific maintenance information in CONTRIBUTING.md, on the other hand, is a common convention (including in this repo).

I would strongly suggest keeping this sentence in CONTRIBUTING.md instead of deleting that file, and then making "the generator" a link to this README. Then this file can describe what exactly the generator does and how to use it.

Reply:
Good suggestions